featurebyte/routes/feature_list/controller.py

```diff
@@ -159,14 +162,19 @@ async def update_feature_list(
 
         Returns
         -------
-        FeatureListModelResponse
-            FeatureList object with updated attribute(s)
+        Union[FeatureListModelResponse, Task]
+            FeatureList object with updated attribute(s) or Task object
+
         """
         if data.make_production_ready:
-            await self.feature_list_facade_service.make_features_production_ready(
+            payload = FeatureListMakeProductionReadyTaskPayload(
                 feature_list_id=feature_list_id,
                 ignore_guardrails=bool(data.ignore_guardrails),
+                user_id=self.service.user.id,
+                catalog_id=self.service.catalog_id,
             )
+            task_id = await self.task_controller.task_manager.submit(payload=payload)
```

Comment:
can potentially inject task manager into this class as well and reference it directly, instead of referencing it via the `task_controller`.